custom/conf/app.ini.sample

```diff
@@ -294,6 +294,12 @@ MIN_PASSWORD_LENGTH = 6
 IMPORT_LOCAL_PATHS = false
 ; Set to true to prevent all users (including admin) from creating custom git hooks
 DISABLE_GIT_HOOKS = false
+;Set password complexity character types
+[security.PASSWORD_COMPLEXITY]
+lower = [a-z]+
```

Comment:
To be honest, this is complex.. I think it should be reverted to the previous one where you could just specify the whole regex in one line.

Reply:
I agree with @adelowo . The setting items are too complex. Maybe we could make it simpler.

Reply:
I apologize, I forgot to fix the app.ini
Fixed in the latest version